src/test/resources/schema/entities.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package org.entities;
+
+import "google/protobuf/descriptor.proto";
+import "schemata_protocol.proto";
+
+option java_package = "org.entities.proto";
+option java_outer_classname = "EntityBuilder";
+
+message Department {
+  int32 id = 1;
+  string name = 2;
+}
+
+message Person {
+  option(org.schemata.schema.message_core).description = "This is the description of the users table";
+  option(org.schemata.schema.message_core).comment = "The comment added after thought";
+  option(org.schemata.schema.owner) = "Growth";
+  option(org.schemata.schema.domain) = "Core";
+  option(org.schemata.schema.type) = ENTITY;
+
+  string name = 1
+  [(org.schemata.schema.field_core).description = "person name"];
```

Comment:
Do we need to enforce a fully qualified name for the metadata-like description? From a user perspective, thinking of how many keystrokes we can reduce.

Reply:
I was thinking about that too, it is quite verbose!

The docs suggest that it can [start from the inner-most package](https://developers.google.com/protocol-buffers/docs/proto3#packages_and_name_resolution), but `protoc` complained about that. I also tried with [`public import` and no package qualifier](https://developers.google.com/protocol-buffers/docs/proto3#importing_definitions), but it didn't like that either.

Unsure if there are some extra protoc options required that aren't obvious in the docs?

Another option may be to just name the package `schemata` without domain or sub-package etc?